apps/admin/frontend/src/screens/unconfigured_screen.tsx

```diff
@@ -314,12 +305,17 @@ export function UnconfiguredScreen(): JSX.Element {
         title={`Convert from ${client?.getDisplayName()} files`}
       >
         <Prose textCenter>
-          <p> Select the following files from a USB drive, etc.</p>
+          <P> Select the following files from a USB drive, etc.</P>
```

Comment:
I know this predates your change, but just want to confirm we still support configuration of these files from non-USB drive storage (files on machine storage? SD card?)

ie. configuration of downstream machines requires a USB drive with signed files but since this UI is for import of SEMS files, requiring signing wouldn't make sense.

Reply:
AFAIK, we do. All of these configuration flows import a file from a path given by the system file picker, which does not restrict paths to the USB drive. I think importing from elsewhere is relevant for development, though, right? As in locked-down production, we wouldn't want someone importing from the machine itself and we wouldn't want to enable SD cards.

Reply:
Yeah, don't have too much context on this flow, but I believe it's supported via USB without requiring the SEMS files to be signed